kuma/wiki/models.py

```diff
@@ -1584,7 +1598,10 @@ class DocumentZone(models.Model):
     attributes inherited by the topic hierarchy beneath it.
     """
     document = models.OneToOneField(Document, related_name='zone')
-    styles = models.TextField(null=True, blank=True)
+    css_filename = models.CharField(
+        max_length=100, null=True, blank=True,
```

Comment:
Agreed, [Django docs](https://docs.djangoproject.com/en/1.8/ref/models/fields/#null) advise to avoid ``null`` on CharFields.